checkov/secrets/scan_git_history.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import git
+import logging
+from typing import TYPE_CHECKING, Dict
+from detect_secrets.core import scan
+
+if TYPE_CHECKING:
+    from detect_secrets import SecretsCollection
+
+
+def get_commits_diff(root_folder: str) -> Dict[str, Dict[str, str]]:
+    commits_diff: Dict[str, Dict[str, str]] = {}
+    try:
+        repo = git.Repo(root_folder)
+    except Exception as e:
+        logging.error(f"Folder {root_folder} is not a GIT project {e}")
+        return commits_diff
+    commits = list(repo.iter_commits(repo.active_branch, max_count=8))
```

Comment:
why only 8 commits?

Reply:
right. I did it for testing and forgot to delete